### PR TITLE
Update args to kwargs

### PR DESCRIPTION
**Pull request by brantleyr**

### CODE REVIEW

File: insights/client/__init__.py [hunk 1]
```diff
@@ -68,94 +68,21 @@ def test_connection(self):
...
-              verbose=False):
+    def fetch(self, **kwargs):
         """
+            parms:
+                core_url (str): URL to retrieve egg from
+                force (bool): force fetch egg (dont check etags)
             returns (str): path to new egg.  None if no update.
         """
         # was a custom egg url passed in?
         if config['core_url']:
```
Comment (kylape) [inline insights/client/__init__.py]: Why not just combine this into one line:

```
egg_url = kwargs.get('core_url', config['core_url'])
```

And just make sure the default in the config for `core_url` is `constants.egg_path`

File: insights/client/__init__.py [hunk 2]
```diff
@@ -68,94 +68,21 @@ def test_connection(self):
...
-            logger.debug('Skipping upload, running offline, or system is not properly registered.')
-            logger.debug('Skipping upload.')
-            logger.debug('Insights results: %s', results)
-            return results
-
-    def fetch(self,
-              egg_url=constants.egg_path,
-              force=False,
-              verbose=False):
+    def fetch(self, **kwargs):
```
Comment (kylape) [inline insights/client/__init__.py]: I think it's more clear to have the kwargs listed out here.  But I don't think the verbose kwarg should be here.  So `fetch(self, egg_url=None, force=False)` sounds reasonable to me.

File: insights/client/__init__.py [hunk 3]
```diff
@@ -281,13 +208,13 @@ def install(self, new_egg):
         logger.debug("The new Insights Core was installed successfully.")
         return {'success': True}
 
-    def update_rules(self, options=None, config=None):
```
Comment (kylape) [inline insights/client/__init__.py]: These kwargs make sense to remove, though.

File: insights/client/__init__.py (hunk 1 above)
Comment (brantleyr) [inline insights/client/__init__.py]: Genius.

**Pull request merged**